_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -181,6 +182,25 @@ UserUnlinked.propTypes = {
 	siteConnected: PropTypes.bool.isRequired,
 };
 
+export class ErrorNoticeCycleConnection extends React.Component {
+	static propTypes = {
+		text: PropTypes.string.isRequired,
```

Comment:
Do you think it's worth setting a default for this?

Reply:
@dereksmart good idea, I'll put some generic text in there.